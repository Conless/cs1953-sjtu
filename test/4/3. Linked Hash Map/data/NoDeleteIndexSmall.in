1
12 1 10 
0 7 mENtsB4Yf0RhOn3yFJag 
0 11 zUGBfc7k9KjNF2OsTyA4 
2 7 
0 11 lLozYtOj2qEGaI5CZQW3 
2 7 
0 9 5kCbx3VZ79vQwE0itfFT 
0 1 XCyB1ta5h8oJT6PfMKEV 
0 11 9TK7Uye6oDXQ0qIziGMW 
0 10 IKnr3Gi2UD9wZ5jgPVF1 
0 7 RlpWIM0SuYAoEC5dsjfy 
