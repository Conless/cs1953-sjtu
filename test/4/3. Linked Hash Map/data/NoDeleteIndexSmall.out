mENtsB4Yf0RhOn3yFJag 
mENtsB4Yf0RhOn3yFJag 
1 XCyB1ta5h8oJT6PfMKEV
7 RlpWIM0SuYAoEC5dsjfy
7 mENtsB4Yf0RhOn3yFJag
9 5kCbx3VZ79vQwE0itfFT
10 IKnr3Gi2UD9wZ5jgPVF1
11 9TK7Uye6oDXQ0qIziGMW
11 lLozYtOj2qEGaI5CZQW3
11 zUGBfc7k9KjNF2OsTyA4
